### PR TITLE
Povilas

### PR DESCRIPTION
**Pull request by PovilasSeibutas**

### CODE REVIEW

File: src/app/app.module.ts [hunk 1]
```diff
@@ -59,8 +60,9 @@ const appRoutes: Routes = [
       },
     ],
   },
-  { path: 'question', component: QuestionComponent },
-  { path: '**', component: NotFoundComponent },
+  { path: "question", component: QuestionComponent },
```
Comment (DariusPa) [inline src/app/app.module.ts]: Is it needed in prod?

File: src/app/app.module.ts [hunk 2]
```diff
@@ -59,8 +60,9 @@ const appRoutes: Routes = [
...
+  { path: "submission", component: SubmissionComponent },
```
Comment (DariusPa) [inline src/app/app.module.ts]: Is it needed in prod?

File: src/app/app.module.ts (hunk 1 above)
Comment (PovilasSeibutas) [reply src/app/app.module.ts]: no

File: src/app/app.module.ts (hunk 2 above)
Comment (PovilasSeibutas) [reply src/app/app.module.ts]: no

**Comment by PovilasSeibutas** [comment]
reoped

**Pull request closed without merging**